config/config.go

```diff
@@ -21,17 +22,24 @@ type Config struct {
 	GitHubOAuthToken *oauth2.Token `json:"github_oauth_token"`
 }
 
-// Path returns either $BUILDKITE_CLI_CONFIG_FILE or ~/.buildkite/config.json
+// Path returns either $BUILDKITE_CLI_CONFIG_FILE, ~/.buildkite/config.json or $XDG_CONFIG_HOME/buildkite/config.json in that order.
 func Path() (string, error) {
 	file := os.Getenv("BUILDKITE_CLI_CONFIG_FILE")
-	if file == "" {
-		home, err := homedir.Dir()
-		if err != nil {
-			return "", err
-		}
+	if file != "" {
+		return file, nil
+
+	}
```

Comment:
Probably best to limit the scope of `file` with
```
if file := os.Getenv("BUILDKITE_CLI_CONFIG_FILE"); file != "" {
	return file, nil
}
```

Reply:
Makes sense, want me to follow up with a PR or will you take care of it?

Reply:
Feel free. We've got a bit on our plates and don't have time to prioritise something like this at the moment.